superset/models/core.py

```diff
@@ -236,6 +237,17 @@ def backend(self) -> str:
         sqlalchemy_url = make_url(self.sqlalchemy_uri_decrypted)
         return sqlalchemy_url.get_backend_name()  # pylint: disable=no-member
 
+    @property
+    def parameters(self) -> Optional[Dict[str, Any]]:
+        # Build parameters if db_engine_spec is a subclass of BasicParametersMixin
+        parameters = {"engine": self.backend}
+
+        if issubclass(self.db_engine_spec, BasicParametersMixin):
```

Comment:
This will not work for BigQuery; we should change it so it works with any DB engine spec that supports the parameters:

```suggestion
        if self.db_engine_spec.parameters_schema is not None:
```